airqo-platform/AirQo-api
# bcc to different emails of inquiries

**_WHAT DOES THIS PR DO?_**

- [x] bcc to different emails of inquiries
- [x] If an email requesting access is an existing user or candidate, do not save that request.

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [AN-243]

**_HOW DO I TEST OUT THIS PR?_**
```
cd src/auth-service
npm install
npm run dev-mac
```
Important to note that we have introduced about 6 new ENVS:
- COMMS_EMAILS=
- POLICY_EMAILS=
- CHAMPIONS_EMAILS=
- RESEARCHERS_EMAILS=
- DEVELOPERS_EMAILS=
- PARTNERS_EMAILS=
- DEFAULT_LIMIT=

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [x] [create inquiry](https://airqo-engineering.postman.co/workspace/internal~7a8e9fad-2d13-4d27-af27-f1f4e9a114bf/request/12513372-298301ab-5236-4fb2-ae0b-2d1556c0b816)
- [x] [request access](https://airqo-engineering.postman.co/workspace/internal~7a8e9fad-2d13-4d27-af27-f1f4e9a114bf/request/12513372-053764bb-d840-4cf8-88d2-ffbb8a347247)

**_SCREENSHOTS?:_** 
<img width="858" alt="Screenshot 2022-12-02 at 04 47 06" src="https://user-images.githubusercontent.com/1590213/205196180-56a3d364-6110-4e84-b77a-a8d549097231.png">
<img width="901" alt="Screenshot 2022-12-02 at 04 47 25" src="https://user-images.githubusercontent.com/1590213/205196192-069b5bd3-0559-423a-b181-32620708d0f5.png">


[AN-243]: https://airqoteam.atlassian.net/browse/AN-243?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
